local_sales/config/database.yml

```diff
@@ -28,6 +28,7 @@ development:
   username: <%= ENV['DB_USERNAME'] %>
   password: <%= ENV['DB_PASSWORD'] %>
 
+
```

Comment:
I recommend you to remove these changes that don't add value to the PR, or you could, you could also think about adding it to the `.gitignore` file
https://stackoverflow.com/a/33484222

Reply:
we use a .ENV to set the settings for our postgre's databse, but i forgot that we used that file (.ENV), and i changed the database.yml, but now i changed the .ENV and removed the database.yml from the .gitignore, cause we have the .ENV in that file

Reply:
@jclopezdev we are using dotenv-rails gem